draft-ietf-dnsop-svcb-https.md

```diff
@@ -763,6 +759,33 @@ in the initial response.  As a performance optimization, if some of the SVCB
 records in the response can be used without requiring additional DNS queries,
 the client MAY prefer those records, regardless of their priorities.
 
+# SVCB-compatible 
+
+An RR type is called "SVCB-compatible" if it permits an implementation that is
+identical to SVCB in its:
+
+* RDATA presentation format
+* RDATA wire format
+* IANA registry used for SvcParamKeys
+* Authoritative server Additional Section processing
+* Recursive resolution process
+* Relevant Class (i.e. Internet ("IN") {{!RFC1035}})
```

Comment:
This section is about defining the word "SVCB-compatible", so the question is: is a definition that allows other classes more useful, or less useful?

I think it's probably less useful, because my goal is that resolvers and authorities should be able to add new SVCB-compatible RR types with a ~1 line change (just add another type number to the "SVCB-compatible types" list).  (And also because other classes no longer exist.)